Add access controls (#20)
cerbos/cloud-docs: modules/ROOT/pages/access-controls.adoc

= Access controls

> In Cerbos Hub, a member can have a role at the organization level and a optional additional roles at the workspace level. The final permission set for the member is the combination of permissions from both roles.

Reviewer: ```suggestion
A Cerbos Hub user can have a role at the organization level and an optional set of roles for each workspace. All of these roles are considered when determining the permissions for any particular user.
```

> == Organization Roles

Reviewer: ```suggestion
== Organization roles
```

> The organization role of a member applies to all workspaces within the organization, except for the `Member` role, which requires a specific workspace-level role assignment to grant any permissions.

Reviewer: ```suggestion
Organization roles, except for the `Member` role, apply to all workspaces within the organization. Users with organizational role of `Member` must be explicitly granted workspace roles in order to access a workspace.
```

[cols="1,1,1,1,1,1", options="header"]
|===
|Action |Owner |Developer |Analyst |Viewer |Member

|View organization
|✅
|✅
|✅
|✅
|✅

|Modify organization
|✅
|❌
|❌
|❌
|❌

|Manage members
|✅
|❌
|❌
|❌
|❌

|Invite a member
|✅
|❌
|❌
|❌
|❌

|Create a workspace
|✅
|✅
|✅
|✅
|✅

|Create a playground
|✅
|✅
|✅
|✅
|✅

|Update a playground
|✅
|✅
|✅
|✅
|✅

|Delete a playground
|✅
|✅
|✅
|✅
|✅

|Export a playground
|✅
|✅
|✅
|✅
|✅

|Connect a PDP to a playground
|✅
|✅
|✅
|✅
|✅
|===

> == Workspace Roles

Reviewer: ```suggestion
== Workspace roles
```

> Permissions assigned at the organization level are inherited by all workspaces. Additionally, a user can be assigned a specific roles within a workspace, potentially granting a higher level of permissions for that workspace.

Reviewer: ```suggestion
Permissions assigned at the organization level are inherited by all workspaces. Additionally, a user can be assigned specific roles within a workspace, potentially granting more permissions for that particular workspace only.
```
